packages/pocket-ic/tests/tests.rs

```diff
@@ -1171,3 +1175,112 @@ fn test_ecdsa_disabled() {
         .unwrap_err();
     assert!(ecdsa_signature_err.contains("Requested unknown or signing disabled threshold key: ecdsa:Secp256k1:dfx_test_key1, existing keys with signing enabled: []"));
 }
+
+#[test]
+fn test_canister_http() {
+    let pic = PocketIc::new();
+
+    // Create a canister and charge it with 2T cycles.
+    let can_id = pic.create_canister();
+    pic.add_cycles(can_id, INIT_CYCLES);
+
+    // Install the test canister wasm file on the canister.
+    let test_wasm = test_canister_wasm();
+    pic.install_canister(can_id, test_wasm, vec![], None);
+
+    // Submit an update call to the test canister making a canister http outcall
+    // and mock a canister http outcall response.
+    let call_id = pic
+        .submit_call(
+            can_id,
+            Principal::anonymous(),
+            "canister_http",
+            encode_one(()).unwrap(),
+        )
+        .unwrap();
+
+    // We need a pair of ticks for the test canister method to make the http outcall
+    // and for the management canister to start processing the http outcall.
```

Comment:
>for the management canister to start processing the http outcall

I think this might be good to know for the user of the outcalls API. Can you add a docstring, perhaps at "get_canister_http"?

Reply:
Added a doc comment.